features/notifications/notifications-init.ts

```diff
@@ -190,16 +189,13 @@ async function maybeDisplayLocalNewMessageNotification(args: {
     messageId: xmtpDecryptedMessage.id,
   })
 
-  // if we're already in the converastion, don't show the notification
-  const currentRoute = getCurrentRoute()
-  if (
-    currentRoute?.name === "Conversation" &&
-    currentRoute?.params?.xmtpConversationId === xmtpConversationId
-  ) {
-    notificationsLogger.debug(
-      "User is already in this conversation, don't display local notification",
-    )
+  // Don't show notifications when app is in foreground
+  const appState = AppState.currentState
+  if (appState === 'active') {
+    notificationsLogger.debug("App is in foreground (active state), don't display local notification")
     return
+  } else {
+    notificationsLogger.debug(`App is not in foreground (${appState}), display local notification`)
```

Comment:
can you use useAppStateStore please and also i don't think we need the debug logs